NDB.Covid19/NDB.Covid19.iOS/Views/InfectionStatus/InfectionStatusViewController.cs

```diff
@@ -316,8 +316,8 @@ async void OnAreYouInfectedBtnTapped(object sender, EventArgs e)
 
         void OpenMessagesPage()
         {
-            UINavigationController vc = MessagePageViewController.GetMessagePageControllerInNavigationController();
```

Comment:
is this method GetMessagePageControllerInNavigationController still needed or we can remove it?

Reply:
It is used in AppDelegate when handling notification tapping, but it might make sense to change it there as well?

Reply:
if we can clean it up a bit it will be great :)

Reply:
Done :)